### DISCUSSION TITLE
Global section under "File Sharering/Samba" the first block "Global" is missing .

### DISCUSSION THREAD

**Discussion by GotraSYS** [cockpit-project/cockpit#21240] [General]
Please I need your help,

I have installed cockpit on an LXC Container in Proxmox. I tested it with Ubuntu and Debian.
But an all I have no Global section under "File Sharering/Samba" the first block "Global" is missing .

I have reinstalled sevarel times but no chance what ever I do No "Global" section . 

Error message that pops up when I click on the link "File Sharing" on the left menu :
_      ParsingError_
__       Error in checkIfSmbConfIncludesRegistry: Invalid INI format at line 0:

Which step do I miss here ?

**Comment by martinpitt** [comment]
Hello @GotraSYS ! This applies to cockpit-file-sharing, which is a completely independent project. Can you please ask this in https://github.com/45Drives/cockpit-file-sharing/discussions ? Thanks!
